--- kBuild-0.1.9998.2700/kBuild/env.sh	2015-03-23 19:32:23.858751190 +0000
+++ kBuild-0.1.9998.2780/kBuild/env.sh	2015-03-23 19:26:51.504892609 +0000
@@ -326,6 +326,9 @@
         x86_64|AMD64|amd64|k8|k8l|k9|k10)
             KBUILD_HOST_ARCH='amd64'
             ;;
+        x32)
+            KBUILD_HOST_ARCH='x32'
+            ;;
         x86|i86pc|ia32|i[3456789]86|BePC)
             KBUILD_HOST_ARCH='x86'
             ;;
