--- kBuild-0.1.9998.3127/kBuild/env.sh.orig	2017-11-25 12:47:06.042644251 +0100
+++ kBuild-0.1.9998.3127/kBuild/env.sh	2017-11-25 12:47:34.349002559 +0100
@@ -356,6 +356,9 @@
                 esac
             fi
             ;;
+        x32)
+            KBUILD_HOST_ARCH='x32'
+            ;;
         x86|i86pc|ia32|i[3456789]86|BePC|i[3456789]86-AT[3456789]86)
             KBUILD_HOST_ARCH='x86'
             ;;
